bin/setup

```diff
@@ -3,7 +3,6 @@ set -euo pipefail
 IFS=$'\n\t'
 set -vx
 
-gem install rake
-bundle install
+[ -n "$CI" ] && bundle config set path 'vendor/bundle'
```

Comment:
I'm sorry I have no idea what this is or how to look it up. Just specifically `[ -n "$CI" ]`.

Reply:
```bash
$ help test
```